flake.nix

```diff
@@ -0,0 +1,46 @@
+{
+  description = "Status Go Flake";
+
+  nixConfig = {
+    extra-substituters = [ "https://nix-cache.status.im/" ];
+    extra-trusted-public-keys = [ "nix-cache.status.im-1:x/93lOfLU+duPplwMSBR+OlY4+mo+dCN7n0mr4oPwgY=" ];
+  };
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/48d567fc7b299de90f70a48e4263e31f690ba03e";
```

Comment:
For the sake of not re-fetching the same stuff multiple times f possible we should ensure we use the same `nixpkgs` across all repos, mobile repo being probably the authoritative source, since it changes rarely and as hard to change:
https://github.com/status-im/status-mobile/blob/ece95ca674c31d696597eccbf700ece8244f12f5/nix/pkgs.nix#L12